packages/react/src/components/Chat/ChatMessage.tsx

```diff
@@ -230,12 +231,9 @@ class ChatMessage extends UIComponent<WithAsProp<ChatMessageProps>, ChatMessageS
       return actionMenuElement
     }
 
-    const menuRect: DOMRect = (positionActionMenu &&
-      _.invoke(this.menuRef.current, 'getBoundingClientRect')) || {
-      height: 0,
-    }
-    const messageRect: DOMRect = (positionActionMenu &&
-      _.invoke(messageNode, 'getBoundingClientRect')) || { height: 0 }
```

Comment:
When we paired with @miroslavstastny we realized that `messageRect.height` is enough in this case